cloudformation/template.yaml

```diff
@@ -126,11 +126,21 @@ Parameters:
       - 'true'
     ConstraintDescription: Malformed input - Parameter DevelopmentMode value must be either 'true' or 'false'
 
+  LocalDevelopmentMode:
+    Type: String
+    Description: Enabling this disables CORS for easier local development. Do not enable this in production! Additionally, do not update a stack that was previously in development mode to be a production stack; instead, make a new stack that has never been in development mode.
+    Default: 'false'
+    AllowedValues: 
+      - 'false'
+      - 'true'
+    ConstraintDescription: Malformed input - Parameter LocalDevelopmentMode value must be either 'true' or 'false'
+
```

Comment:
1. `LocalDevelopmentMode` is basically the same as `DevelopmentMode` on the front end stuff (no CORS, allows localhost), but without the rest of the backend reconfiguration for development mode (S3 set to release mode, etc.).
2. Not sure, but I plan to look into it later. There's only a handful of resources `DevelopmentMode` can affect that `LocalDevelopmentMode` doesn't, and it's fairly easy to figure out which ones based on the conditions set in the template.

Reply:
FWIW, I don't want `LocalDevelopmentMode` to stick around permanently, either, if it helps. It's an ugly hack and the only reason I created it was because `DevelopmentMode` was giving me errors that didn't exist in release, but not enabling it gave me CORS issues when I was using the `localhost` dev server.